src/calculator.rs

```diff
@@ -10,15 +10,19 @@
 // express or implied. See the License for the specific language governing permissions and
 // limitations underthe License.
 
+//! calculator module
+//!
+//! Provides Calculator struct for parsing string expressions to floats.
+
```

Comment:
Could you move the documentation as a `///` style documentation to lib.rs where mod calculator is imported. That is considered preferable by rust conventions afaik.